src/Console/Command/FixCommand.php

```diff
@@ -156,6 +156,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
         ;
 
         if (null !== $stdErr) {
+            if (OutputInterface::VERBOSITY_VERBOSE <= $output->getVerbosity()) {
```

Comment:
maybe we should add it to all commands? (of course only for verbose output and only to stdErr)
eg, `describe` output is also different for different PHP CS Fixer version.

Reply:
:+1: